packages/connect/src/connect-error.spec.ts

```diff
@@ -137,6 +163,14 @@ describe("ConnectError", () => {
       expect(got.cause).toBe(error);
     });
   });
+  describe("instanceof", () => {
+    it("works for the actual ConnectError", () => {
+      expect(new ConnectError("foo")).toBeInstanceOf(ConnectError);
+    });
```

Comment:
Let's add tests for `null instanceof ConnectError === false`, `undefined instanceof ConnectError === false`, and `"err" instanceof ConnectError === false`